Depends-On: 0003_win32_msi_filename.patch
Touch-Files: src/win32/tip/tip_resource.rc
Scope: ui

diff --git a/src/win32/tip/tip_resource.rc b/src/win32/tip/tip_resource.rc
index ea09b1459..0897a9e31 100644
--- a/src/win32/tip/tip_resource.rc
+++ b/src/win32/tip/tip_resource.rc
@@ -123,7 +123,7 @@ END
   #if defined(GOOGLE_JAPANESE_INPUT_BUILD)
     #define MOZC_RES_FILE_DESCRIPTION "Google 日本語入力 TIP モジュール"
   #elif defined(MOZC_BUILD)
-    #define MOZC_RES_FILE_DESCRIPTION "Mozc TIP Module"
+    #define MOZC_RES_FILE_DESCRIPTION "Mozc4med TIP Module"
   #endif
 #endif
 #ifndef MOZC_RES_FILETYPE
@@ -145,8 +145,8 @@ BEGIN
     IDS_IME_DISPLAYNAME     "Google 日本語入力"
     IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Google 日本語入力"
 #else
-    IDS_IME_DISPLAYNAME     "Mozc"
-    IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Mozc"
+    IDS_IME_DISPLAYNAME     "Mozc4med"
+    IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Mozc4med"
 #endif
     IDS_INPUTMODE           "入力モード"
     IDS_WIN8_TRAY_ITEM      "右クリックで IME のオプションを表示します"
@@ -161,7 +161,7 @@ BEGIN
 #if defined(GOOGLE_JAPANESE_INPUT_BUILD)
     IDS_ABOUT               "Google 日本語入力について"
 #else
-    IDS_ABOUT               "Mozc について"
+    IDS_ABOUT               "Mozc4med について"
 #endif
     IDS_FULL_KATAKANA       "全角カタカナ"
     IDS_HALF_ALPHANUMERIC   "半角英数"
@@ -175,10 +175,10 @@ BEGIN
     IDS_CANDIDATE_WINDOW             "Google 日本語入力 候補ウィンドウ"
     IDS_INDICATOR_WINDOW             "Google 日本語入力 インジケータ ウィンドウ"
 #else
-    IDS_UNOBSERVABLE_SUGGEST_WINDOW  "Mozc サジェストウィンドウ"
-    IDS_OBSERVABLE_SUGGEST_WINDOW    "Mozc サジェスト候補ウィンドウ"
-    IDS_CANDIDATE_WINDOW             "Mozc 候補ウィンドウ"
-    IDS_INDICATOR_WINDOW             "Mozc インジケータ ウィンドウ"
+    IDS_UNOBSERVABLE_SUGGEST_WINDOW  "Mozc4med サジェストウィンドウ"
+    IDS_OBSERVABLE_SUGGEST_WINDOW    "Mozc4med サジェスト候補ウィンドウ"
+    IDS_CANDIDATE_WINDOW             "Mozc4med 候補ウィンドウ"
+    IDS_INDICATOR_WINDOW             "Mozc4med インジケータ ウィンドウ"
 #endif
 END
 
@@ -188,8 +188,8 @@ BEGIN
     IDS_IME_DISPLAYNAME     "Google Japanese Input"
     IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Google Japanese Input"
 #else
-    IDS_IME_DISPLAYNAME     "Mozc"
-    IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Mozc"
+    IDS_IME_DISPLAYNAME     "Mozc4med"
+    IDS_TEXTSERVICE_DISPLAYNAME_SYNONYM  "Mozc4med"
 #endif
     IDS_INPUTMODE           "Input Mode"
     IDS_WIN8_TRAY_ITEM      "Right click to show IME options"
@@ -204,7 +204,7 @@ BEGIN
 #if defined(GOOGLE_JAPANESE_INPUT_BUILD)
     IDS_ABOUT               "About Google Japanese Input"
 #else
-    IDS_ABOUT               "About Mozc"
+    IDS_ABOUT               "About Mozc4med"
 #endif
     IDS_FULL_KATAKANA       "Full-width Katakana"
     IDS_HALF_ALPHANUMERIC   "Half-width Alphanumeric"
@@ -217,9 +217,9 @@ BEGIN
     IDS_OBSERVABLE_SUGGEST_WINDOW    "Google Japanese Input Suggestion Candidate Window"
     IDS_CANDIDATE_WINDOW             "Google Japanese Input Candidate Window"
 #else
-    IDS_UNOBSERVABLE_SUGGEST_WINDOW  "Mozc Suggestion Window"
-    IDS_OBSERVABLE_SUGGEST_WINDOW    "Mozc Suggestion Candidate Window"
-    IDS_CANDIDATE_WINDOW             "Mozc Candidate Window"
+    IDS_UNOBSERVABLE_SUGGEST_WINDOW  "Mozc4med Suggestion Window"
+    IDS_OBSERVABLE_SUGGEST_WINDOW    "Mozc4med Suggestion Candidate Window"
+    IDS_CANDIDATE_WINDOW             "Mozc4med Candidate Window"
 #endif
 END
 
